File: springSecurityCasDemo/src/main/webapp/js/service/userService.js
//服务层
app.service('userService',function($http){
	    	
	//增加 
	this.addUser=function(entity){
		return  $http.post('../user/add.do',entity );
	}
	
	this.getCode=function(phone){
		return  $http.get('../user/getCode.do?phone='+phone);
	}

});

File: youlexuan-user-interface/src/main/java/com/offcn/service/UserService.java
package com.offcn.service;

import java.util.List;

import com.offcn.pojo.TbAddress;
import com.offcn.pojo.TbUser;

public interface UserService {

	public void addUser(TbUser user);
	
	public List<TbUser> findAll();
	
	public List<TbAddress> findAddressByUserName(String userId);
	
}
